### ISSUE TITLE
[BUG] Services related to deployment/statefulset are not displayed on rancher UI

### ISSUE THREAD

**Issue by WeAreHadock** [rancher/dashboard#8826]

**Comment by skanakal** [comment]
This issue reproducible with rancher 2.7.3 & 2.7.2, and **ui-dashboard-index=2.7.2**  and it works when I change ui-dashboard-index and ui-index to 2.7.1.

![image](https://github.com/rancher/dashboard/assets/124564266/110502f8-4a6b-4ddc-9867-f2f291f0e5dd)

**Comment by ly5156** [comment]
This issue should have been fixed by this [PR](https://github.com/rancher/dashboard/pull/8722)

Because the user did not install monitoring, in the workload details page, the fetch method called the allDashboardsExist(shell/utils/grafana.js) method, which called the `await store.dispatch(`${ storeName }/find`, { type: CATALOG.APP, id: 'cattle-monitoring-system/rancher-monitoring' })` method, because it did not add a try catch, resulting in the exception thrown to interrupt the logic behind, and the matchingServices property could not be initialized


<img width="1317" alt="image" src="https://github.com/rancher/dashboard/assets/3406205/1bedec44-03d2-4ac8-bbaf-70bd2a00f857">

**Comment by skanakal** [comment]
@israel-brain Can you open a new issue with minimal reproducer steps?

**Comment by israel-brain** [comment]
@skanakal 
Yes, I’ll open a ticket with the minimal steps to reproduce. Thanks!
